tests/functional/object/mcg/test_write_to_bucket.py

```diff
@@ -456,3 +459,42 @@ def test_content_encoding_with_write(
         logger.info(
             "Put object operation is preserving ContentEncoding as a object metadata"
         )
+
+    @bugzilla("2259189")
+    @bugzilla("2264480")
+    @pytest.mark.polarion_id("OCS-5773")
+    def test_nb_db_activity_logs_on_io(
+        self, bucket_factory, awscli_pod_session, mcg_obj
+    ):
+
+        """
+        This test checks if the activity logs are being logged
+        in the activitylogs table for every object upload and
+        deletion when noobaa log is set to default
+
+        """
+
+        bucket = bucket_factory()[0]
+        full_objectpath = f"s3://{bucket.name}"
+        logger.info("successfully created bucket")
+
+        sync_object_directory(
+            awscli_pod_session, AWSCLI_TEST_OBJ_DIR, full_objectpath, mcg_obj
+        )
+        logger.info("uploaded objects to the bucket")
+
+        rm_object_recursive(awscli_pod_session, bucket.name, mcg_obj)
+        logger.info("deleted all the objects from the bucket")
+
+        nb_activitylogs = exec_nb_db_query("SELECT data ->> 'event' FROM activitylogs;")
+        logger.info("successfully fetched noobaa db activitylogs data")
+
+        assert "obj.uploaded" not in str(
+            nb_activitylogs
+        ), "Object upload event is being logged in activitylogs table"
+        assert "obj.deleted" not in str(
+            nb_activitylogs
+        ), "Object deletion event is being logged in activitylogs table"
+        logger.info(
+            "No object upload/deletion info is being updated in the activitylogs table"
+        )
```

Comment:
there is no need to check for the DB activitylogs size?

Reply:
Nope. we only need to make sure that the activity logs doesn't include the updates for simplest operations like object upload/delete when the log level is at default mode. size of the logs can be subjective and there is no parameter to benchmark it.